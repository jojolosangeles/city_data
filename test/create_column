set -x
python ../code/main.py scripts/create_column.txt > actual/create_column.out
diff expected/create_column.out actual/create_column.out
diff expected/sample20.csv actual/sample20.csv
