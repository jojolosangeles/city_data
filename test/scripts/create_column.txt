sample20 = data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv
sample20.columns
sample20.latitude <= [eval(x)[0] for x in dataFrame['Location']]
sample20.longitude <= [eval(x)[1] for x in dataFrame['Location']]
sample20.columns
