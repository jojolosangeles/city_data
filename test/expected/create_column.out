Test: scripts/create_column.txt
> sample20 = data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv
Loaded data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv into DataFrame 'sample20', 19 rows
> sample20.columns
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> sample20.latitude <= [eval(x)[0] for x in sample20['Location']]
> sample20.longitude <= [eval(x)[1] for x in sample20['Location']]
> sample20.columns
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location' 'latitude'
 'longitude']
> sample20.save
Saved CSV File './actual/sample20.csv'
